src/main.rs

```diff
@@ -65,7 +66,7 @@ pub async fn execute_subcommand(command: Option<Command>) -> ResultWithDefaultEr
                 billable,
                 description,
                 project: _,
-            } => StartCommand::execute(get_api_client()?, billable, description).await?,
+            } => StartCommand::execute(get_api_client()?, description, billable).await?,
```

Comment:
Do you think we should accept a opts struct here (similar to what's described in #45)
```rust
struct StartCommandOptions {
  workspace_id: Option<u64>,
  project_id: Option<u64>,
  task_id: Option<u64>,
  description: String,
  tags: Vec<String>,
  billable: bool,
}
```

That way we can reuse `StartCommand` from more places and avoid a huge list of parameters for each property the time-entry creation API accepts?